### PR TITLE
clenaup 2.0: remove s_finding_severity_naming

### LINKED ISSUE #3707: remove s_finding_severity_naming feature flag
I think @mtesauro mentioned this was an old artifact from the days of yore?

### PR DESCRIPTION
**Pull request by manuel-sommer**
remove s_finding_severity_naming fixes #3707

### CODE REVIEW

**Review by damiencarol — CHANGES_REQUESTED** [review]

File: dojo/templatetags/display_tags.py
Comment (manuel-sommer) [reply dojo/templatetags/display_tags.py]: @damiencarol thanks, good catch. is it fine like this?

**Comment by manuel-sommer** [comment]
@damiencarol Could you review it?

**Review by valentijnscholten — APPROVED** [review]
approved, but the migration needs a rename and dependency update after some recent other merges into dev.

File: dojo/db_migrations/0072_remove_system_settings_s_finding_severity_naming.py
Comment (damiencarol) [inline dojo/db_migrations/0072_remove_system_settings_s_finding_severity_naming.py:9]: there is a need to change that as we merged some PR with migration (see the current number in dev branch)

**Review by damiencarol — CHANGES_REQUESTED** [review]

**Comment by valentijnscholten** [comment]
@manuel-sommer we have decided to go ahead with this removal from the UI parts. Could you rebase to the latest dev or merge latest dev into your branch? And rename/update your migration accordingly? We will merge soon afterwards.

**Comment by valentijnscholten** [comment]
superseded by #4415 due to lack of response and no access for maintainers to update the PR

**Pull request closed without merging**
